### PR TITLE
[Term Entry] Java ArrayList: .get()

### LINKED ISSUE #1441: [Term Entry] Java ArrayList: `.get()`
### Reason/inspiration (optional)

We'd like to add a new term entry for the `.get()` method under the Java: ArrayList concept. In the new entry, please include the following:

- A "Syntax" section with a \```pseudo code snippet.
- An "Example" section with a \```java code snippet.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by hectorconte**
### Description

Updated with some minor changes, but created a new pull request since the previous one was unintentionally deleted.

Closes #1441 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by hectorconte** [comment]
Sorry @HishamT, @caupolicandiaz, I accidentally deleted the previous pull request, but made the reviewed changes here. Sorry again.

**Review by HishamT — APPROVED** [review]
LGTM!

File: content/java/concepts/array-list/terms/get/get.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+Title: '.get()'
+Description: 'Retrieves the element present in a specified position in an ArrayList.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
```
Comment (caupolicandiaz) [inline content/java/concepts/array-list/terms/get/get.md:9]:
```suggestion
  - 'Methods'
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@hectorconte Looks good, just added a substitute for one of the tags that wasn't really relevant to this entry

**Comment by hectorconte** [comment]
@caupolicandiaz Great, done!

**Review by caupolicandiaz — APPROVED** [review]
@hectorconte Nice, good to go

**Comment by yangc95** [comment]
@hectorconte live link: https://www.codecademy.com/resources/docs/java/array-list/get

It looks like you're currently an anonymous contributor:
Screen Shot 2023-02-02 at 6 46 22 PM

If you'd like to become a public contributor, make sure to add your GitHub username to your Codecademy profile and set your visibility to "everyone." 🙂

cc: @HishamT @caupolicandiaz

**Comment by hectorconte** [comment]
@yangc95 I do have set it up that way, strange! Do you think I should contact codecademy support maybe? Thanks Chris!

**Comment by yangc95** [comment]
@hectorconte Try signing out and logging back in with the GitHub option!

**Comment by hectorconte** [comment]
@yangc95 That did it! Thanks again!

**Pull request merged**